content/support/changelog/peer-to-peer/index.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: layout-sidebar
+name: Peer-to-Peer (Beta)
+description: Information on additions and changes for the Peer-to-Peer API (Beta).
+order: 100
+published: true
+showInNav: true
+icon: fa fa-share-alt
+back_to_top: true
+title: Peer-to-Peer (Beta) Changelog
+---
+
+# {{ name }}
+
+Monitor this page to keep up with the [Peer-to-Peer API (Beta)]({{ stache.config.peer_to_peer_entity_reference }}) latest changes and {{ stache.config.api_type_name }} service releases.
+
+## 2017-03-28
+
+### New
+
+The Peer-to-Peer API has been released for a public beta. This API is used to create supporter pages, and query campaigns, donors, donations, and teams.
+
+For more information, check out the [entity]({{ stache.config.peer_to_peer_entity_reference }}) and [endpoint]({{ stache.config.portal_endpoints_peer_to_peer }}) references.
```

Comment:
In the fundraising and gift APIs that we just released in beta, we have a sentence that says "The initial release contains endpoints to ..." and provide a few examples of what the endpoints can do. Do we have anything similar we can add here? Like maybe "The initial release contains endpoints to access data and perform operations such as searching for records and updating information."?